rules/naming/tests/Rector/Assign/RenameVariableToMatchGetMethodNameRector/Fixture/clousure.php.inc

```diff
@@ -20,13 +22,15 @@ class ClosureClass
 
 namespace Rector\Naming\Tests\Rector\Assign\RenameVariableToMatchGetMethodNameRector\Fixture;
 
+use DateTime;
+
 class ClosureClass
 {
     public function run()
     {
         $datetime = new DateTime();
         array_map(function ($element) use ($datetime) {
-            $timestamp = $datetime->getTimestamp() + $element;
+            $timestamp = $datetime->getTimestamp();
```

Comment:
This case shoud be skipped, as:

```php
apple + banana != 'apple'
```

It creates something new, so we can't guess name of the new thing